src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -169,9 +169,28 @@ public void aliasTest() throws IOException {
 
     @Test
     public void groupByTest() throws Exception {
-
         JSONObject result = executeQuery(String.format("SELECT COUNT(*) FROM %s/account GROUP BY gender",
-                TEST_INDEX_ACCOUNT));
+            TEST_INDEX_ACCOUNT));
+        assertResultForGroupByTest(result);
+    }
+
+    @Test
+    public void groupByUsingTableAliasTest() throws Exception {
+        JSONObject result = executeQuery(String.format("SELECT COUNT(*) FROM %s/account a GROUP BY a.gender",
+            TEST_INDEX_ACCOUNT));
+        assertResultForGroupByTest(result);
+    }
+
+    @Test
+    public void groupByUsingTableNamePrefixTest() throws Exception {
+        JSONObject result = executeQuery(String.format(
+            "SELECT COUNT(*) FROM %s/account GROUP BY elasticsearch-sql_test_index_account.gender",
```

Comment:
np: `"SELECT COUNT(*) FROM %s/account GROUP BY %s.gender", TEST_INDEX_ACCOUNT , TEST_INDEX_ACCOUNT `

Reply:
I'd like to keep this one for now because I also added other similar test cases. Replacing actual index name with %s is okay here but will end up with many %s for others.